jsettlers.mapcreator/src/jsettlers/mapcreator/main/action/AbortDrawingAction.java

```diff
@@ -24,6 +24,10 @@
  *
  */
 public class AbortDrawingAction extends Action {
+
+	/**
+	 * Constructor TODO NOT CALLED
+	 */
```

Comment:
I add a new documentation, michael said I should call it on ESC press, but I haven't the time to do this, I have first to look how the key eventhandling is implemented